netbox/core/models/contenttypes.py

```diff
@@ -21,6 +21,24 @@ def public(self):
             q |= Q(app_label=app_label, model__in=models)
         return self.get_queryset().filter(q)
 
+    def with_feature(self, feature):
```

Comment:
Not a part of this PR, but I think we should have a FR to document the registry as it is getting a bit complicated now, for example for a model-feature what it is for, you need to do the following steps (like in the clean validate the content-type), etc...

Reply:
We have a page in [the developer docs](https://docs.netbox.dev/en/stable/development/application-registry/) that explains each store. Happy to flesh this out some. (And your comment reminded me to go add the new `tables` store introduced under #14173.)